tests/my_test.spec.js

```diff
@@ -0,0 +1,26 @@
+const { test, expect } = require("@playwright/test");
+const { beforeEach } = require("node:test");
```

Comment:
you are not using this import. beforeEach is the method of **test**. So can delete this line